lib/facter/mcollective_configured.rb

```diff
@@ -0,0 +1,5 @@
+Facter.add('mcollective_configured') do
```

Comment:
Would this fact better be called `mcollective_installed`? Existence of the server config only shows that mco exists, not that it was ever set up to do anything